SentinelHub/tests/test_utils_geo.py

```diff
@@ -0,0 +1,69 @@
+# coding=utf-8
+from typing import Tuple, TypeVar
+
+import pytest
+
+pytest.importorskip("qgis.core")
+
+from qgis.core import QgsRectangle  # noqa: E402
+
+from ..settings import Settings  # noqa: E402
+from ..utils.geo import bbox_to_string, get_custom_bbox, is_bbox_too_large, is_supported_crs  # noqa: E402
+
+T = TypeVar("T", float, int)
+
+
+@pytest.mark.parametrize(
+    "lat_max, lat_min, lng_max, lng_min, coords",
+    [
+        ("1", "0", "1", "0", (0, 0, 1, 1)),
+        ("0", "0", "0", "0", (0, 0, 0, 0)),
+        (
+            "2.0",
+            "1.0",
+            "8.0",
+            "9.0",
+            (8, 1, 9, 2),
```

Comment:
if you remove the comma at the end of this line `black` will format it as a one-liner
If you do `[a, b, c,]` then it will split it into multiple lines even if it can fit into one.

Reply:
good point